.circleci/config.yml

```diff
@@ -491,7 +491,7 @@ jobs:
       - run:
           name: Licences check
           command: |
-            bundle exec rake ci:license_finder:run
+            bundle exec rake license_finder:compliance
```

Comment:
Calling `compliance` instead of `run` skips the call to `report` when `compliance` fails, is that intended?

Reply:
We will keep the report in the repo. What I don't see is why circleci should generate it again? The job is to _check_ the licenses, not generating a new report. Am I missing something else?